src/app/pages/website/blog/blog.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'redirect',
+  template: 'redirecting...'
```

Comment:
Suggestion: It would be nice to show this text in a centered layout with some HTML. I assume this is just going ot stick at the top left of the page.